src/containers/ReactivesearchContainer.js

```diff
@@ -9,88 +9,56 @@ import {
-              />
-              <MultiList
-                componentId="Creator"
-                dataField="creator.label.keyword"
-                title="Creator"
-                showSearch={false}
-                URLParams={true}
-              />
-              <MultiList
-                componentId="Contributor"
-                dataField="contributor.label.keyword"
-                title="Contributor"
-                showSearch={false}
-                URLParams={true}
-              />
-              <MultiList
-                componentId="Genre"
-                dataField="genre.label.keyword"
-                title="Genre"
-                showSearch={false}
-                URLParams={true}
-              />
-              <MultiList
-                componentId="Language"
-                dataField="language.label.keyword"
-                title="Language"
-                showSearch={false}
-                URLParams={true}
-              />
-              <MultiList
-                componentId="AdminSet"
-                dataField="admin_set.title.keyword"
-                title="Library Unit"
-                showSearch={false}
-                URLParams={true}
-              />
-              <MultiList
-                componentId="RightsStatement"
-                dataField="rights_statement.label.keyword"
-                title="Rights Statement"
-                showSearch={false}
-                URLParams={true}
-              />
-              <MultiList
-                componentId="StylePeriod"
-                dataField="style_period.label.keyword"
-                title="Style Period"
-                showSearch={false}
-                URLParams={true}
-              />
-              <MultiList
-                componentId="Subject"
-                dataField="subject.label.keyword"
-                title="Subject"
-                showSearch={false}
-                URLParams={true}
-              />
-              <MultiList
-                componentId="Technique"
-                dataField="technique.label.keyword"
-                title="Technique"
-                showSearch={false}
-                URLParams={true}
-              />
-              <MultiList
-                componentId="Visibility"
-                dataField="visibility.keyword"
-                title="Visibility"
-                showSearch={false}
-                URLParams={true}
-              />
+              {facets.map(facet => (
+                <MultiList
+                  componentId={facet.name.replace(/\s+/g, '')}
+                  dataField={facet.field}
+                  title={facet.name}
+                  showCheckbox={false}
+                  showMissing={true}
+                  showSearch={false}
+                  URLParams={true}
+                  react={{
+                    and: allFilters.filter(entry => {
+                      return entry !== facet.name.replace(/\s+/g, '');
+                    })
```

Comment:
`react` prop is some kind of Reactive Search magic here which applies the filtering config?